From ce17830d1feefc8735d3dcef884b4f632b4d7c6f Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Tue, 16 Sep 2014 20:19:24 -0400
Subject: [fs] nfsd: drop unused stp arg to alloc_init_deleg

Message-id: <1410898849-22126-81-git-send-email-bfields@redhat.com>
Patchwork-id: 95759
O-Subject: [Patch RHEL7 080/165] nfsd: drop unused stp arg to alloc_init_deleg
Bugzilla: 1044798 1123721 1141932
RH-Acked-by: Benjamin Coddington <bcodding@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Jeff Layton <jlayton@primarydata.com>

Signed-off-by: Jeff Layton <jlayton@primarydata.com>
Reviewed-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: f9416e281e53bea6f8e39c21f50fd79c029ba24a
Bugzilla: 1044798
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4state.c b/fs/nfsd/nfs4state.c
index b421b51..049ef2c 100644
--- a/fs/nfsd/nfs4state.c
+++ b/fs/nfsd/nfs4state.c
@@ -579,7 +579,7 @@ static void block_delegations(struct knfsd_fh *fh)
 }
 
 static struct nfs4_delegation *
-alloc_init_deleg(struct nfs4_client *clp, struct nfs4_ol_stateid *stp, struct svc_fh *current_fh)
+alloc_init_deleg(struct nfs4_client *clp, struct svc_fh *current_fh)
 {
  struct nfs4_delegation *dp;
  long n;
@@ -3649,7 +3649,7 @@ nfs4_open_delegation(struct net *net, struct svc_fh *fh,
   default:
    goto out_no_deleg;
  }
- dp = alloc_init_deleg(oo->oo_owner.so_client, stp, fh);
+ dp = alloc_init_deleg(oo->oo_owner.so_client, fh);
  if (dp == NULL)
   goto out_no_deleg;
  status = nfs4_set_delegation(dp, stp->st_file);
-- 
1.7.1